src/gui/main_gui.cpp

```diff
@@ -32,16 +33,21 @@ GUITest(World world) {
 
     auto mesh_trees = entity::generate_mesh(default_trees_voxel);
 
+    //glEnable( GL_DEBUG_OUTPUT );
+    //glDebugMessageCallback( MessageCallback, 0 );
+
```

Comment:
Delete

Reply:
This was the think I wanted you to impliment.